api/v1alpha1/clientprofile_types.go

```diff
@@ -31,10 +31,15 @@ type CephFsConfigSpec struct {
 
 	//+kubebuilder:validation:Optional
 	FuseMountOptions map[string]string `json:"fuseMountOptions,omitempty"`
+
+	//+kubebuilder:validation:XValidation:rule="self == oldSelf",message="radosNamespace is immutable"
```

Comment:
Why the immutability?

Reply:
https://github.com/ceph/ceph-csi-operator/pull/165#discussion_r1828962621